README.md

```diff
@@ -17,22 +18,6 @@
   - [Node.js](#nodejs)
   - [Go](#go)
 - [API](#api)
-  - [Files](/API/files)
-    - [`add`](/API/files#add)
-    - [`createAddStream`](/files#createaddstream)
-    - [`cat`](/API/files#cat)
-  - [Object](/API/object)
-    - [`object.new`](/API/object#objectnew)
-    - [`object.put`](/API/object#objectput)
-    - [`object.get`](/API/object#objectget)
-    - [`object.data`](/API/object#objectdata)
-    - [`object.links`](/API/object#objectlinks)
-    - [`object.stat`](/API/object#objectstat)
-    - [`object.patch`](/API/object#objectpatch)
-      - [`object.patch.addLink`](/API/object#objectpatchaddlink)
-      - [`object.patch.rmLink`](/API/object#objectpatchrmlink)
-      - [`object.patch.appendData`](/API/object#objectpatchappenddata)
-      - [`object.patch.setData`](/API/object#objectpatchsetdata)
```

Comment:
They are not in the file. The Table of Contents should reflect the current file.